boards/atlflight/eagle/qurt-default.cmake

```diff
@@ -10,6 +10,7 @@ if (DEFINED ENV{QC_SOC_TARGET})
 else()
 	set(QC_SOC_TARGET "APQ8074")
 endif()
+add_definitions(-D__PX4_QURT_EAGLE_${QC_SOC_TARGET})
```

Comment:
Do you need this specific definition? I would think you can just use `__PX4_QURT_EAGLE`:
https://github.com/PX4/Firmware/blob/490543045795b2282a47b949a18b943ccf2a2186/boards/atlflight/eagle/qurt-default.cmake#L40

Reply:
Actually I was hoping to get rid of these defines entirely. We should have the build system always set something for every board, rather than do it manually in each board config.

Reply:
Let's do that after this PR I'd say.

Reply:
Removed the define see comment bellow.